test/functional/dataElements/xdmObject/helpers/xdmTree.js

```diff
@@ -42,57 +42,78 @@ const getIsElementInViewport = selector => {
   )();
 };
 
+const create = node => {
```

Comment:
I made this into a function so that the next() method could work on this object.